src/ReportManager/SchemaForm/SchemaFormContext/index.test.js

```diff
@@ -0,0 +1,226 @@
+import React, { useContext } from 'react';
+import { render, screen } from '../../../test-utils';
+
+import SchemaFormContextProvider, { SchemaFormContext } from './index';
+import userEvent from '@testing-library/user-event';
+import { FORM_FIELDS_TYPES } from '../constants';
+
+describe('ReportManager - SchemaForm - SchemaFormContext', () => {
+
+  const efb = {
```

Comment:
Lets change the names of these to schema. `efb` is not appropriate since that is not what it is.

Reply:
I left all feedback of test out just to wait for your approval on the second iteration, I'm working on fixing tests rn